checkbox-ng/plainbox/impl/execution.py

```diff
@@ -713,3 +729,53 @@ def get_execution_command(
         cmd += ["aa-exec", "-p", "unconfined"]
     cmd += [job.shell, "-c", job.command]
     return cmd
+
+
+def get_execution_command_systemd_unit(
+    job, environ, session_id, nest_dir, target_user, extra_env=None
+):
+    """
+    Generates a command that if executed runs a Checkbox command section
```

Comment:
This docstring header is exactly the same as the one for `get_execution_command_subshell`. Maybe


```suggestion
    Generates a command that if executed runs a Checkbox command section as a systemd unit
```

or something like that (not sure how to keep it under 79 chars while differentiating it from the other one though)